designs/administrator/future-openshift/cluster-settings/index.md

```diff
@@ -0,0 +1,64 @@
+
+![Channel help text](img/channel-help.png)
+- Clicking `Channel` surfaces a popover that provides context as to what a channel is along  with a link to documentation to learn more about channels in OpenShift.
+
+![Channel visualization](img/channel-visualization.png)
+- The blue line signifies the current channel you are on. The current version and recommended version (if available) are both marked with blue dots. 
+- The gray line signifies the next available channel.
+- To surface recommended versions in the next available channel, users must switch to the channel by clicking the `Pencil` next to `Channel`. 
+- If no other channels are available the gray line will not be visible. 
+- At this time the visualization only supports surfacing one additional channel - the next available.
+
+![Channel visualization - recommended version popover](img/recommended-channel.png)
+- Clicking on the recommended version, in this case the `4.3.22 dot`, will surface a popover with recommended minor release information and a direct link to release notes.
+
+![Channel visualization - “+ more” popover](img/more-channels.png)
+- Clicking `+ More` will surface a popover that lists out all versions (between the current version and recommended version) in a table with direct links to release notes. 
+
+![Channel visualization - “+ more” modal](img/modal.png)
+- In the case that there are more than 5 versions available (between the current version and recommended version), clicking `+ More` will surface a modal.
+
+## Updating a Channel and Viewing the In Progress Checklist
+
+It was important to provide more transparency to the user when an update is in progress. The mockups below illustrate the steps a user takes to update to a minor release in their current channel, and surfaces the in progress checklist that allows users to see the status of the update in real time.
+
+![Update to minor release](img/update.png)
+- Clicking `Update` launches a modal to confirm the update.
+
+![Update modal](img/update-modal.png)
+- The recommended version is preselected. 
+- If available, users can select an alternative version in the dropdown.
+- Clicking `Update` will begin the update.
+
+![In progress check list](img/check-list.png)
+- The channel visualization is replaced with an in progress checklist including: Cluster Operators, Master Nodes and Worker Nodes. 
+- Each item contains a corresponding count (ex. 2 of 3), a status icon, and a loader to provide more context to the user. 
+- Clicking `Cluster Operators` will navigate users to the cluster operators list. 
+- Clicking `Nodes` will navigate users to the nodes list to view detailed progress of each node.
+
+![Nodes list](img/nodes.png)
+- The nodes list can be sorted by `Status`. 
+- Additional context has been added under “Ready”. The node will display “Up to date” or “Waiting on update” below the existing “Ready” status, to reflect the status of that particular node. 
```

Comment:
Does the new additional “Up to date” or “Waiting on update” status always appear on nodes now? Or only when a cluster update operation is happening?

Reply:
I believe the thought was that “Up to date” or “Waiting on update” would only appear during the actual update.

Reply:
Cool, makes sense.